deployment/pom.xml

```diff
@@ -79,36 +78,19 @@
     </dependency>
     <dependency>
       <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-netty-deployment</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.testcontainers</groupId>
-      <artifactId>cassandra</artifactId>
-      <version>1.12.4</version>
-      <exclusions>
```

Comment:
Exclusions not required IMO, the tests run fine without them.

Reply:
Same situation, banned dependencies:
https://github.com/datastax/cassandra-quarkus/issues/79#issuecomment-646476395